Included additional responses from Vivien + Richard (#6)
greenelab/nature_news_manuscript: content/05.discussion.md

Previous anecdotal studies from journalists have shown that awareness of their bias can help them to reduce it [@https://medium.com/ladybits-on-medium/i-analyzed-a-year-of-my-reporting-for-gender-bias-and-this-is-what-i-found-a16c31e1cdf; @https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
Once a bias is identified an individual can seek resources to help them find and retain diverse sources, such as utilizing international expert databases like gage [@https://gage.500womenscientists.org/] and SheSource [@https://www.womensmediacenter.com/shesource/].
Additional tips for journalists to achieve and maintain a diverse source pool is described by Christina Selby in the Open Notebook [@https://www.theopennotebook.com/2016/08/23/including-diverse-voices-in-science-stories/].

It should also be mentioned that we were only able to analyze the data provided through scraping "www.nature.com".

> This is a major limitation, because the only measures that we have of demographics of sources are persons who have their name mentioned within the article or cited in the article.

Reviewer: ```suggestion
This is a major limitation, because the only measures that we have of demographics of sources are persons who have their name mentioned or research cited within the article.
```

> This is only one aspect of the journalistic research process; a journalist may talk to several diverse persons, read their work, but not cite or mention them for several possible reasons.

Reviewer: ```suggestion
Journalists do not quote or mention all of the sources that they interviewed or cite all of the papers that they read when researching an article.
```

> For example, a person may not be directly included in the article because of journalisitc style, that a person does not want to be included, or that they only provide informational content that is not directly quotable, but instead shapes the content of the article.

Reviewer: ```suggestion
For example, a person may not be mentioned or quoted included in the article because of length limitations, because they do not want to be named, or if they provide information that is not directly quotable but that still shapes the content of the article.
```

> A more accurate reflection of journalists' sources would be a self-maintained record of persons they interview, to ensure any bias is limited.

Reviewer: ```suggestion
A more accurate reflection of journalists' sources would be a self-maintained record of persons they interview.
```

> However, it is still useful to identify possible biases in which demographics tend to receive more recognition within journalistic output, which can be measured by mentions, quotes, or citations of persons.

Reviewer: ```suggestion
Our work examines disparities with respect to recognition within stories, which can be measured by mentions, quotes, or citations of persons.
```

> Furthermore, many journalists are limited by the persons who can respond to their requests for an interview or leads from prominent scientists.

Reviewer: ```suggestion
Furthermore, many journalists are limited by who responds to their requests for an interview or guided by recommendations from prominent scientists.
```

> Scientists can also audit themselves to ensure that not only are they referring field experts to journalists, but are also not allowing their biases to influence the experts they suggest. 

Reviewer: ```suggestion
Scientists fielding reporter inquiries can also audit themselves to examine the extent to which there are disparities in the sets of experts they recommend. 
```

<!-- CUT Furthermore, since news coverage is accountable to their readership, it presents the opportunity to represent scientific perspectives that are more diverse than observed in academic publishing. 
Through the use of one measure of source demographics, we have shown that gender parity can be achieved in at least one column type, as observed by the gender parity in quotes from the "Career Features" column.!-->

> Journalists and referee scientists have a unique opportunity to shape the public and their peers' perspectives on who is a scientific expert.

Reviewer: ```suggestion
Journalists and the scientists they interview have a unique opportunity to shape the public and their peers' perspectives on who is a scientific expert.
```

> Their choice of coverage topics and interviewees could help to diminish possible biases in scientific journalism.

Reviewer: ```suggestion
Their choice of coverage topics and interviewees could help to reduce disparities in the outputs of science-related journalism.
```
